src/averageAges.js

```diff
@@ -36,8 +45,23 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
-function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+function calculateWomenAverageAge(people, withChildren = false) {
+  if (withChildren === false) {
```

Comment:
Consider using a ternary operator instead of if-else to make the code more concise